### ISSUE TITLE
Recovered Mesh from Rotation in processed_motion

### ISSUE THREAD

**Issue by yw0208** [tr3e/InterGen#34]
Hi, I tried to visualize human mesh according to rotation in _npy_ file of processed_motions.
This is what I got:
![image](https://github.com/tr3e/InterGen/assets/70508717/28da7597-be65-4984-afcd-0030c6f5a00b)
```
    motions = np.load("InterHuman/motions_processed/person1/1.npy")
    motions = torch.from_numpy(motions)
    motions=motions[:,62*3: 62*3+21*6].reshape(-1, 21, 6)
    motions=rotation_6d_to_matrix(motions)
    motions=matrix_to_axis_angle(motions)
    zero=torch.zeros(motions.shape[0], 1, 3)
    motions=torch.cat([zero, motions, zero, zero], dim=1)
    motions=np.array(motions)[0].reshape(-1)
    joblib.dump(motions, 'scratch/temp/intergen_obj/pro_test.pt')
```
I get rotation like this, and input the 72-dim vector as pose to the SMPL model.
Do you have any advice to fix it?

**Comment by Heibaiii** [comment]
> Hi, I tried to visualize human mesh according to rotation in _npy_ file of processed_motions. This is what I got: ![image](https://github.com/tr3e/InterGen/assets/70508717/28da7597-be65-4984-afcd-0030c6f5a00b)
> 
> ```
>     motions = np.load("InterHuman/motions_processed/person1/1.npy")
>     motions = torch.from_numpy(motions)
>     motions=motions[:,62*3: 62*3+21*6].reshape(-1, 21, 6)
>     motions=rotation_6d_to_matrix(motions)
>     motions=matrix_to_axis_angle(motions)
>     zero=torch.zeros(motions.shape[0], 1, 3)
>     motions=torch.cat([zero, motions, zero, zero], dim=1)
>     motions=np.array(motions)[0].reshape(-1)
>     joblib.dump(motions, 'scratch/temp/intergen_obj/pro_test.pt')
> ```
> 
> I get rotation like this, and input the 72-dim vector as pose to the SMPL model. Do you have any advice to fix it?

HI, i have the same question, have you solved it?
Can you tell me how to solve it?
Thanks!
